index.html

```diff
@@ -349,60 +390,65 @@ <h3 class="language-header">Lanuages</h3>
-          If you have an application you are interested in developing, a feature
-          that you need built or a project that needs coding. I’d love to help
-          with it
-        </p>
-        <form action="https://formspree.io/f/xeqnnvdb" method="post">
-          <ul class="contact-ul">
-            <li class="form-list">
-              <input
-                type="text"
-                class="input-text"
-                placeholder="John Doe"
-                name="user_name"
-                required
-                maxlength="30"
-              />
-            </li>
-            <li class="form-list">
-              <input
-                type="email"
-                class="input-text"
-                required
-                placeholder="example@gmail.com"
-                name="user_email"
-              />
-            </li>
-            <li class="form-list">
-              <textarea
-                class="contact-text-area"
-                id="msg"
-                name="user_message"
-                maxlength="500"
-                required
-                placeholder="Leave a message here"
-              ></textarea>
-            </li>
-            <li>
-              <button class="contact-button" type="submit">Get in touch</button>
-            </li>
-          </ul>
-        </form>
+
+        <!-- contact section -->
+        <section id="contact-me">
+          <div class="contact-container">
+            <h1 class="contact-header">Contact me</h1>
+            <p class="contact-paragraph">
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+            <form action="https://formspree.io/f/xeqnnvdb" method="post">
+              <ul class="contact-ul">
+                <li class="form-list">
+                  <input
+                    type="text"
+                    class="input-text"
+                    placeholder="John Doe"
+                    name="user_name"
+                    required
+                    maxlength="30"
+                  />
+                </li>
+                <li class="form-list">
+                  <input
+                    type="email"
+                    class="input-text"
+                    required
+                    placeholder="example@gmail.com"
+                    name="user_email"
+                  />
+                </li>
+                <li class="form-list">
+                  <textarea
+                    class="contact-text-area"
+                    id="msg"
+                    name="user_message"
+                    maxlength="500"
+                    required
+                    placeholder="Leave a message here"
+                  ></textarea>
+                </li>
+                <li class="contact-button-li">
+                  <button class="contact-button" type="submit">
```

Comment:
[OPTIONAL] For the `:hover` styling I recommend you add a border since the button's background color is the same as the section background color and it looks a little strange to hover over it and have it melt into the background.